ttnn/cpp/ttnn/operations/full_like/device/full_like_factory.cpp

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include "common/constants.hpp"
+#include "full_like_device_operation.hpp"
+#include "host_api.hpp"
+#include "impl/buffers/circular_buffer_types.hpp"
+#include "ttnn/tensor/tensor_utils.hpp"
+#include "ttnn/tensor/types.hpp"
+#include "tt_metal/common/work_split.hpp"
+
+namespace ttnn::operations::full_like {
+
+using namespace tt;
+using namespace tt::tt_metal;
+using namespace tt::constants;
+
+
+union datatype {
+    uint32_t u32;
+    float f32;
+} u;
```

Comment:
I think you should rename this to u_fill_value

Reply:
Sure thing homie